core/extensions/extension.hpp

```diff
@@ -101,6 +101,16 @@ namespace kagome::extensions {
                                  runtime::WasmPointer value_data,
                                  runtime::WasmSize value_length) = 0;
 
+    /**
+     * @brief Reads data from storage with the given key
+     * @param key pointer-size to the key
+     * @param value_out pointer-size to the read data
+     * @param offset in bytes from the data block begin should be read
+     */
+    virtual runtime::WasmSpan ext_storage_read(runtime::WasmSpan key,
```

Comment:
![image](https://user-images.githubusercontent.com/9370151/87425821-f9a96680-c5e6-11ea-8b21-c6e948e320ed.png)

In spec it is called `ext_storage_read_version_1`. Please rename here and in other places